doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -859,6 +859,77 @@ However elimination to `Type` or to a polymorphic sort with `s := Prop` is allow
         : Type@{s|max(u,v)}
         := pair { pr1 : A; pr2 : B pr1 }.
 
+Explicit Sorts
+---------------
+
+Similarly to universes, fresh global sorts can be declared with the :cmd:`Sort`.
+
+.. cmd:: Sort {+ @ident }
+         Sorts {+ @ident }
+
+   In the monomorphic case, declares new global sort qualities 
+   with the given names.  Global quality names live in their own namespace.
+   Inside sections, the command respects the `Universe Polymorphism` flag,
+   either set globally or locally through the :attr:`universes(polymorphic)` attribute (or
+   the ``Polymorphic`` legacy attribute), meaning the sort
+   quantification will be discharged for each section definition
+   independently. Polymorphic sort qualities are forbidden outside sections.
+
+  .. exn:: Polymorphic sorts can only be declared inside sections, use #[universe(polymorphic=no)] Sort instead.
+
+    This error spwans when a :cmd:`Sort` command is issued outside a section with universe polymorphism set. 
+    Either scope the :cmd:`Sort` with an englobing :cmd:`Section` or be explicit about creating a global sort
+    by turning universe polymorphism off.
+
+  .. exn:: Cannot declare global sort qualities in a module type.
+
+    The :cmd:`Sort` command is not supported in a module type.
+
+.. cmd:: Print Sorts
+   :name: Print Sorts
+
+   Print the list of global named sorts in the current context. Use :cmd:`Show Universes` to print sort variables local to a proof.
```

Comment:
"current context" meaning the proof context or the containing section?

If Print Sorts and Show Universes are complementary, maybe this command should be "Show Sorts"?  Too bad we haven't been consistent in using "Show" or "Print" for various commands.

Should there be a way to print the local named sorts in the current context?

Reply:
`Print Sorts` is the sort equivalent of `Print Universes`, meaning that it prints the global sort environment. `Show Universes` prints both universe and sort variables local to a proof context.